skipper.go

```diff
@@ -802,6 +811,20 @@ func createDataClients(o Options, auth innkeeper.Authentication) ([]routing.Data
 		}
 	}
 
+	if o.RemoteRoutesFile != "" {
+		for _, rf := range strings.Split(o.RemoteRoutesFile, ",") {
```

Comment:
Lets check if we can use `commaListFlag` instead

Reply:
Sure. I missed that piece. Thanks.